imports/ui/components/navbar.jsx

```diff
@@ -10,18 +9,24 @@ class NavbarWrapper extends React.Component {
 
   render() {
     return (
-      <React.Fragment>
-        <Navbar className="navbar navbar-expand-lg container">
-          <Navbar.Brand href="/">
-             CodeBuddies Connect
-           </Navbar.Brand>
-          <Navbar.Toggle aria-controls="basic-navbar-nav"/>
-            <Nav className="mr-auto"></Nav>
-            <Nav.Link className="mr-sm-2" href="/faq">FAQ</Nav.Link>
-            <Button variant="outline-success" className="btn btn-primary btn-sm btn-border">Login</Button>
-          </Navbar>
-          {this.props.children}
-      </React.Fragment>
+      <Container>
+        <Navbar>
+          <Link to="/" className='navbar-brand'>
```

Comment:
@lpatmo Made some tweaks. Using anchor tag directly for routing causes the page to reload.